### DISCUSSION TITLE
Podman Desktop v1.7.0 is available 🎉!

### DISCUSSION THREAD

**Discussion by deboer-tim** [podman-desktop/podman-desktop#5695] [Announcements]
A new release of 🦭 Podman Desktop is out: [v1.7.0](https://github.com/containers/podman-desktop/milestone/27?closed=)

![renovations](https://github.com/containers/podman-desktop/assets/19958075/86d92c30-432b-43a5-a0df-bd1005cef192)

Download it from the [download section](https://podman-desktop.io/downloads) of the [Podman Desktop website](https://podman-desktop.io/)

Release Notes are available at https://podman-desktop.io/blog/podman-desktop-release-1.7

Full Changelog is available at [1.7.0](https://github.com/containers/podman-desktop/releases/tag/v1.7.0).

Thank you very much for your support and your contributions!! 🎉

**Comment by marindobeka** [comment]
The current download link from the [download section](https://podman-desktop.io/downloads) refers to version 1.6.2, not 1.7.0.
See the following screenshot:
<img width="535" alt="image" src="https://github.com/user-attachments/assets/a6ff8fb5-551e-4135-b64e-6bc59488c282" />

**Reply by slemeur** [reply]
Hi @marindobeka. Yeah the website always points to the latest version available of Podman Desktop. We recommend you using the latest version. In case you need a prior version, you should find them on GitHub: https://github.com/podman-desktop/podman-desktop/releases

**Reply by benoitf** [reply]
1.16.2 is a more recent version than 1.7.0

16 `>` 7
